src/Attribute/Controller/Id.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types = 1);
+
+namespace Apitte\Core\Attribute\Controller;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_CLASS)]
+final class Id
+{
+
+	/** @var string */
+	private $name;
+
+	public function __construct(string $name)
+	{
+		$this->name = $name;
+	}
+
+	public function getName(): string
+	{
+		return $this->name;
+	}
+
+}
```

Comment:
With array-based doctrine annotations that contain logic in the __construct(), it's better to use a new class only for attributes. 

Instead of mixing 2 classes at once with 2 different consturctors, we have a strict typed parameters. We can clearly state what parameters are required and which optional :+1: